lib/utils.js

```diff
@@ -34,17 +19,6 @@ class Utils {
   static unsupportedEnvironment() {
     return Promise.reject(new Error('Unsupported environment. This method only works on the server (node.js).'));
```

Comment:
Change error. "For a browser-cmpatible version, go to https://www.npmjs.com/package/homefront".